src/adservice/Dockerfile

```diff
@@ -11,6 +11,9 @@ RUN ./gradlew installDist
 
 FROM openjdk:8-alpine
 
+RUN apk update ; \
```

Comment:
thanks for your PR, can we make this:

    apk add --no-cache libc6-compat

doing apk update adds a package cache to the resulting image.

Reply:
Updated PR